testing/notifier-interactions.md

```diff
@@ -1,3 +1,8 @@
+# Testing
+
+Most of us are familiar with unit testing code with more traditional concurrency models. In this case of threaded code, for many, this means throwing up your hands in disgust and walking away. Testing concurrency with threads is hard. Testing concurrency is Pony is much easier but if you haven't encountered anything like it before it can be hard to know where to start.
```

Comment:
"Testing concurrency is Pony is much easier" -> "Testing concurrency in Pony is much easier"